lib/radius-rails.rb

```diff
@@ -1 +1,9 @@
 require "radius/rails"
+
+module RadiusRails
+  class Railtie < Rails::Railtie
+    config.app_generators do |g|
+      g.templates.unshift File::expand_path('../templates', __FILE__)
```

Comment:
Our standard is to use dot `.` for class / module method calls and `::` for constant references. Also, any reason we need to support a Ruby version that doesn't have `__dir__`? I believe all of our apps are on a version which support it.

```suggestion
      g.templates.unshift File.expand_path('templates', __dir__)
```